packages/ember-toucan-form/package.json

```
@@ -29,19 +29,23 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config",
     "format:check": "prettier -c .",
-    "format": "prettier -w ."
+    "format": "prettier -w .",
+    "_syncPnpm": "pnpm sync-pnpm"
   },
   "peerDependencies": {
+    "@crowdstrike/ember-toucan-core": "workspace:*",
```

Since this is a peer dependency, I feel like this should be a version specified for consumers rather than what I have added here.  So if someone installs `@crowdstrike/ember-toucan-form`, we also want them to be on some version of `@crowdstrike/ember-toucan-core` as well as that'll be used.  With this syntax though, I'm not sure how that'll quite work since this is an add-on consuming yet another add-on.

I feel like this should be the following instead:

```suggestion
    "@crowdstrike/ember-toucan-core": "^0.1.0",
```

But then how about the `devDependency` below?  Should that remain `workspace:*` with the pnpm dependency syncing?  I haven't done something like this in a monorepo before so I'm a bit out of my comfort zone.

---

> I feel like this should be the following instead:

Right! `workspace:*` is a pnpm specific thing AFAIK, so we cannot expose this to consumers. It should be what you suggested. Same [here](https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/changeset/package.json#L32).

> But then how about the devDependency below? Should that remain workspace:* with the pnpm dependency syncing?

I used the same specifier [here](https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/changeset/package.json#L43). Pnpm will still use the code from the repo rather than pulling the actually published version from npm, as long as the versions match. At least that's my understanding. But `workspace:*` will _guarantee_ that you will always get the code from the repo, and _never_ pull from npm. Which is right I think. So we could try that for our devDependencies, and see how it goes...

---

Nice, thank you!  I'll go ahead and get this updated.  I'll give it a shot in devDependencies with `workspace:*` and see what happens